presto-main/src/main/java/io/prestosql/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1278,14 +1278,27 @@ private void analyzeFiltersAndMasks(Table table, QualifiedObjectName name, Optio
                     .withRelationType(RelationId.anonymous(), new RelationType(fields))
                     .build();
 
+            Set<String> allColumns = fields.stream()
+                    .filter(field -> field.getName().isPresent())
```

Comment:
You can replace those two lines with:
`flatMap(field -> field.getName().stream())